homeassistant/components/avri/sensor.py

```diff
@@ -0,0 +1,105 @@
+"""Support for Avri waste curbside collection pickup."""
+import logging
+
+from avri.api import Avri, AvriException
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+CONF_COUNTRY_CODE = "country_code"
+CONF_POSTCODE = "postcode"
+CONF_HOUSE_NUMBER = "house_number"
+CONF_HOUSE_NUMBER_EXTENSION = "house_number_extension"
+DEFAULT_NAME = "avri"
+ICON = "mdi:trash-can-outline"
+SCAN_INTERVAL = 14_400
```

Comment:
Not sure if the scan_interval can contain underscores.
```suggestion
SCAN_INTERVAL = 14400
```

Reply:
Since PEP 515 (https://www.python.org/dev/peps/pep-0515/) any numeric literal can contain underscores.

However, to avoid confusion and improve readability I made it a `timedelta` as other components also seem to use that :).

Reply:
Thanks did not know that :)

That is even better 👍 FYI you will need to run `isort` to fix the pylint error